### PR TITLE
unpublish/cancel article once it has been killed

### PR DESCRIPTION
**Pull request by takeit**
| Q                         | A
| ------------------------- | ---
| Bug fix?                  | no
| New feature?              | yes
| BC breaks?                | no
| Deprecations?             | no
| Tests pass?               | yes
| Changelog update required | no
| Fixed tickets             | SWP-567
| License                   | AGPLv3

- [x] - add tests

### CODE REVIEW

File: docs/content_publishing.rst [hunk 1]
```diff
@@ -50,3 +50,9 @@ How to check (in code) if content is published?
 .. note::
 
     Content assigned to routes is automatically checked (if it's publishable for current user) by event listener.
+
+
+How to Un-publish Content?
+--------------------------
+
+If an article is published you can easily un-publish it via API as described above. Another way to un-publish already published article is by ``killing`` the article. It can be achieved by setting the value of ``pubStatus`` property to ``canceled`` in the JSON (Ninjs) content according to `IPTC standards <http://cv.iptc.org/newscodes/pubstatusg2/canceled>`_. Once that status will be set, and the content will be send to Publisher (/api/v1/content/push API endpoint), article will be un-published immediately and its status will be set to ``canceled``.
```
Comment (ahilles107) [inline docs/content_publishing.rst]: don't keep api prefix and version in docs just provide: `/content/push`

File: src/SWP/Bundle/BridgeBundle/Resources/config/serializer/Model.BaseContent.yml [hunk 2]
```diff
@@ -63,3 +63,7 @@ SWP\Component\Bridge\Model\BaseContent:
         keywords:
             expose: true
             type: array
+        pubStatus:
```
Comment (ahilles107) [inline src/SWP/Bundle/BridgeBundle/Resources/config/serializer/Model.BaseContent.yml]: why not `publicationStatus`?

Comment (takeit) [inline src/SWP/Bundle/BridgeBundle/Resources/config/serializer/Model.BaseContent.yml]: I keep here the same convention as it is in the ninjs (ninjs validator) format according to IPTC and the way Superdesk handles this.

File: docs/content_publishing.rst (hunk 1 above)
Comment (takeit) [inline docs/content_publishing.rst]: done

**Pull request merged**